modules/storage-api/src/testFixtures/java/org/apache/ignite/internal/storage/index/AbstractSortedIndexStorageTest.java

```diff
@@ -280,20 +275,15 @@ void testSingleColumnIndex(ColumnDefinition columnDefinition) throws Exception {
      */
     @Test
     void testPutIdempotence() throws Exception {
-        List<ColumnDefinition> columns = List.of(
-                column(ColumnType.string().typeSpec().name(), ColumnType.string()).asNullable(false).build(),
-                column(ColumnType.INT32.typeSpec().name(), ColumnType.INT32).asNullable(false).build()
-        );
-
         SortedIndexDefinition indexDefinition = SchemaBuilders.sortedIndex("TEST_INDEX")
-                .addIndexColumn(columns.get(0).name()).asc().done()
-                .addIndexColumn(columns.get(1).name()).asc().done()
+                .addIndexColumn(ColumnType.string().typeSpec().name()).asc().done()
```

Comment:
Why don't you use a constant, like `ColumnTypeSpec.STRING`?

Reply:
Fix it